### PR TITLE
Fixed JUSTIN callback with updated wsdl

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

JUSTIN callback wsdl was out of date.
- updated wsdl,
- enhanced logging.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

mvn test

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/resources/DocumentDistributionMainProcess.xsd [hunk 1]
```diff
@@ -44,14 +44,17 @@
...
-				<element name="jobId" type="string"/>
-                <element name="statusCode" type="string"/>
-                <element name="statusMessage" type="string"/>
-			</sequence>
-		</complexType>
-	</element>
+			<complexType>
+				<sequence>
+					<element name="jobId" type="string"/>
+	                <element name="statusCode" type="string"/>
```
Comment (akroon3r) [inline src/main/resources/DocumentDistributionMainProcess.xsd:50]: The diff says that lines 50-51 are indented at a different level here compared to <----. Is that supposed to be the case?

**Review by BrendanBeachBC — APPROVED** [review]
Looks good to me!

**Comment by akroon3r** [comment]
I saw you updated that one log line from TRACE to DEBUG. Did we update those other log lines from DEBUG to INFO?

File: src/main/resources/DocumentDistributionMainProcess.xsd (hunk 1 above)
Comment (KDNeufeld) [reply src/main/resources/DocumentDistributionMainProcess.xsd:50]: yeah, I just fixed the indentation on a couple lines.  The main change is lines 56-57

**Comment by KDNeufeld** [comment]
@akroon3r , I didn't yet, no.  We were going to confirm which lines we need in PROD.  Right now, most things are DEBUG.  We can promote the desired log lines to INFO - can do this in a separate PR.

**Pull request merged**